src/App.js

```diff
@@ -0,0 +1,12 @@
+import {Header} from './Header'
+import {DisplayButtons} from './DisplayButtons'
```

Comment:
Format the destructuring
```suggestion
import { Header } from './Header'
import { DisplayButtons } from './DisplayButtons'
```